tests/yast2_gui/yast2_firewall_settings.pm

```diff
@@ -0,0 +1,114 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: YaST2 Firewall UI test checks verious configurations and settings of firewall
+# Make sure yast2 firewall can opened properly. Configurations can be changed and written correctly.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_guitest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use network_utils 'iface';
+use YaST::Module;
+use version_utils "is_sle";
```

Comment:
seems that the only part missing is to break this module in several ones, avoiding verification in UI and making it really simple as start/stop ones.

Reply:
https://openqa.opensuse.org/tests/2809501  TW
https://openqa.suse.de/t9742845  SLE